graylog2-web-interface/package.json

```diff
@@ -67,15 +67,17 @@
     "babel-plugin-react-transform": "^1.1.1",
     "clean-webpack-plugin": "^0.1.3",
     "css-loader": "^0.23.0",
-    "enzyme": "^1.4.1",
+    "enzyme": "^2.0.0",
     "eslint": "^2.2.0",
-    "eslint-config-airbnb": "6.0.2",
+    "eslint-config-airbnb": "^9.0.0",
```

Comment:
What about spending more amount of work updating it and disable the rules that do not make sense?